config/samples/flows_v1beta2_flowcollector.yaml

```diff
@@ -22,6 +22,13 @@ spec:
       interfaces: []
       excludeInterfaces: ["lo"]
       kafkaBatchSize: 1048576
+      #flowFilter:
+      #  config:
+      #    action: Accept
+      #    cidr: 0.0.0.0/0
+      #    protocol: UDP
+      #    sourcePorts: 53
+      #  enable: true
```

Comment:
That's awesome ! 

Do you think we should write some doc or maybe a blog to highlight real use cases ?

Reply:
I did add a doc in ebpf agent with usecases that can become blog if we need to